documentation/docs/getting_started/using_docker.md

```diff
@@ -7,40 +9,42 @@ keywords:
 - Run
 - macOS
 - Windows
-description: Install and run a Hornet node using Docker.  Recommended for macOS and Windows.
-image: /img/logo/HornetLogo.png
+- how to
+
 ---
 
 # Using Docker
 
-Hornet Docker images (amd64/x86_64 architecture) are available at [gohornet/hornet](https://hub.docker.com/r/gohornet/hornet) Docker hub.
+Hornet Docker images (amd64/x86_64 architecture) are available at the [gohornet/hornet](https://hub.docker.com/r/gohornet/hornet) Docker hub.
 
 ## Requirements
 
-1. A recent release of Docker enterprise or community edition. You can finde installation instructions in the [official Docker documentation](https://docs.docker.com/engine/install/).
-2. [GIT](https://git-scm.com/)
-3. [CURL](https://curl.se/)
-4. At least 1GB available RAM
+1. A recent release of Docker enterprise or community edition. You can find installation instructions in the [official Docker documentation](https://docs.docker.com/engine/install/).
+2. [GIT](https://git-scm.com/).
+3. [CURL](https://curl.se/).
+4. At least 1GB available RAM.
 
 ## Clone the Repository
 
-Once you have completed all the installation [requirements](#requirements), you can clone the repository or by running:
+Once you have completed all the installation [requirements](#requirements), you can clone the repository by running:
 
 ```sh
 git clone https://github.com/gohornet/hornet && cd hornet && git checkout mainnet
 ```
 
-:::info
+:::note
+
 The next portion of the guide assumes you are executing commands from the root directory of the repository.
+
 :::
 
 ## Prepare
 
 1. If you want to use alternative ports, edit the `config.json` file.
 
-2. Add your neighbors addressed to the `peering.json` file.
+2. Add your neighbors addresses to the `peering.json` file.
 
-The Docker image runs under user with user id 65532 and group id 65532. To make sure there are no permission issues you will need to:
+The Docker image runs under "user" with user id 65532 and group id 65532. To make sure there are no permission issues, you will need to:
```

Comment:
why quotation marks?

Reply:
So they wouldn't be confused it was referring to _them_ as a user. But I'll swap it back.